test/mocha/27-http-edv-client-update.js

```diff
@@ -119,5 +118,27 @@ describe('bedrock-edv-storage HTTP API - edv-client update', () => {
         err.message.should.equal(
           '"sequence" is too large.');
       });
+    it('should increase sequence when updating a deleted document',
+      async () => {
+        await edvClient.insert({
+          doc: mockData.httpDocs.alpha,
+          invocationSigner: capabilityAgent.getSigner(),
+        });
+        const doc = await edvClient.get({
+          id: mockData.httpDocs.alpha.id,
+          invocationSigner: capabilityAgent.getSigner(),
+        });
+        // delete doc
+        await edvClient.delete({
+          doc,
+          invocationSigner: capabilityAgent.getSigner()});
+        // get doc
+        const record = await edvClient.get({
```

Comment:
If I understand correctly, the `get` API here is happily fetching a document that has been marked as deleted.  Is that right?  If that's true, as consumers of an EDV, we must *always* do a check on every document that we `get` to ensure that it is not deleted?  Seems like a `NotFoundError` thrown by the client would be appropriate when trying to get a deleted document unless some override flag was supplied to the `get` API.

Reply:
Yes, it was decided here and on the [edv-client](https://github.com/digitalbazaar/edv-client/pull/71) that deleting means removing the content, adding a `deleted: true` under the `meta` property.

Reply:
@mandyvenables thanks for the clarification.  @dlongley @gannan08 @aljones15 Do we need a patch to the edv-client `get` API to avoid unexpected behavior?

Reply:
I filed an issue for that here: https://github.com/digitalbazaar/edv-client/issues/75 <-- please discuss over there and then someone can make a PR if we decide we want an error thrown.

Reply:
Sounds good.